kafka/src/kafka/impl/configuration.cpp

```diff
@@ -16,13 +16,22 @@
 #include <userver/yaml_config/yaml_config.hpp>
 
 #include <kafka/impl/error_buffer.hpp>
+#include <kafka/impl/log_level.hpp>
 
 USERVER_NAMESPACE_BEGIN
 
 namespace kafka::impl {
 
 namespace {
 
+/// @brief Redirect `librdkafka` logs to `userver` logs.
+///
+/// @see
+/// https://docs.confluent.io/platform/current/clients/librdkafka/html/rdkafka_8h.html#a06ade2ca41f32eb82c6f7e3d4acbe19f
+void KafkaLogCallback([[maybe_unused]] const rd_kafka_t*, int level, const char* fac, const char* buf) noexcept {
+    LOG(userver::kafka::impl::convertRdKafkaLogLevelToLoggingLevel(level)) << fac << buf;
```

Comment:
please, add at least space between facillity (fac) and message (buf)

better variant is `std::format("facility: {}, message: {}", facility, message)`

Reply:
Added fmt::format in case someone uses C++ less than the 20th standard

Reply:
sorry, it was missclick) i implied fmt::format of course